From 398f7f851d61cd33543089e41f4280f9815ec01e Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Fri, 5 Sep 2014 11:04:02 -0400
Subject: [ethernet] sfc: Adding PCI ID for Solarflare 7000 series 40G network adapter

Message-id: <1409915048-26438-27-git-send-email-nikolay@redhat.com>
Patchwork-id: 91425
O-Subject: [RHEL7.1 PATCH BZ 1110888 26/32] sfc: Adding PCI ID for Solarflare 7000 series 40G network adapter.
Bugzilla: 1110888
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mateusz Wrzesinski <mwrzesinski@solarflare.com>

Signed-off-by: Shradha Shah <sshah@solarflare.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 3b06a00e65fbb5c526371143beaaa2221d39d577)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/sfc/efx.c b/drivers/net/ethernet/sfc/efx.c
index a6b3972..1e987cd 100644
--- a/drivers/net/ethernet/sfc/efx.c
+++ b/drivers/net/ethernet/sfc/efx.c
@@ -2607,6 +2607,8 @@ static DEFINE_PCI_DEVICE_TABLE(efx_pci_table) = {
   .driver_data = (unsigned long) &siena_a0_nic_type},
  {PCI_DEVICE(PCI_VENDOR_ID_SOLARFLARE, 0x0903),  /* SFC9120 PF */
   .driver_data = (unsigned long) &efx_hunt_a0_nic_type},
+ {PCI_DEVICE(PCI_VENDOR_ID_SOLARFLARE, 0x0923),  /* SFC9140 PF */
+  .driver_data = (unsigned long) &efx_hunt_a0_nic_type},
  {0}   /* end of list */
 };
 
-- 
1.7.1